conbench/entities/benchmark_result.py

```diff
@@ -125,7 +125,7 @@ def create(data):
             repository = repository_to_url(data["github"]["repository"])
 
         # create if not exists
-        commit = Commit.first(sha=sha, repository=repository)
+        commit = Commit.first(sha=sha, repository={repository.lower() if repository else None})
```

Comment:
I don't think you need the set
```suggestion
        commit = Commit.first(
            sha=sha, repository=repository.lower() if repository else None
        )
```

Reply:
Thank you! I have no idea what I was thinking.